src/gt4py/definitions.py

```diff
@@ -143,27 +143,27 @@ def __or__(self, other):
 
     def __lt__(self, other):
         """Element-wise comparison."""
-        return self._compare(self._broadcast(other), operator.lt)
+        return self._compare(self._broadcast(other), operator.lt, any)
```

Comment:
I think there is a bug here because this reduction does not check that elements that are not lower are also not larger, so it will give the wrong result if one of the elements is lower and the other are larger (e.g. `(1, 3) < (0, 50) == True`)

Reply:
Thanks! I implemented the changes we discussed.